product_docs/docs/eprs/7/05_smr_operation/08_optimizing_performance/02_optimize_sync_replication/using_bacth_lob.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Using batch synchronization for LOB objects"
+---
+
+When synchronizing tables with LOB-type columns (with BLOB, CLOB, BYTEA etc. objects), the Replication Server copies the rows by grouping 5 rows in a batch by default. This is to avoid causing out-of-memory errors when an LOB column has a large amount of data. You can customize the batch size to increase or decrease the number of rows copied in a batch.
```

Comment:
Remove "the" article from "... , the Replication Server copies ...".

Reply:
```suggestion
When synchronizing tables with LOB-type columns (with BLOB, CLOB, BYTEA etc. objects), Replication Server copies the rows by grouping 5 rows in a batch by default. This is to avoid causing out-of-memory errors when an LOB column has a large amount of data. You can customize the batch size to increase or decrease the number of rows copied in a batch.
```